app/index.js

```diff
@@ -188,9 +188,23 @@ app.on('ready', () => {
         appActions.newWindow()
       }
     } else {
-      loadedPerWindowImmutableState.forEach((wndState) => {
-        appActions.newWindow(undefined, undefined, wndState)
-      })
+      loadedPerWindowImmutableState
```

Comment:
Maybe as a follow up, it would be good to pull out the sorting into something which can easily be unit tested.  Maybe a new method `determineZindex` or similar (maybe in `app/commmon/state/windowState`)

then the below would be like:
```
const sortedList = windowState.determineZindex(loadedPerWindowImmutableStore)
sortedList.forEach((wndState) => {
  appActions.newWindow(undefined, undefined, wndState)
})
```

Reply:
++!